cpp/ql/src/Likely Bugs/Conversion/ArrayArgSizeMismatch.ql

```diff
@@ -5,6 +5,7 @@
  * @kind problem
  * @id cpp/array-arg-size-mismatch
  * @problem.severity warning
+ * @precision medium
```

Comment:
What's preventing this query from having higher precision? If it's considered as a `recommendation` instead of a `warning`, could it then have `@precision very-high`?

If it only deserves `@precision medium`, I'd expect to see a test case with `// GOOD [FALSE POSITIVE]`.

Reply:
I'll upgrade it to `warning high`.  That means it will be displayed on LGTM by default, though it only has one or two results per 10 projects anyway.  I don't think I can be confident it's `very-high` without having seen many more results.